gui/src/store/modules/user-login.ts

```diff
@@ -49,6 +50,10 @@ export default class UserLogin extends VuexModule {
   public setUserPermissions(permissions: any) {
     this.userPermissions = permissions;
   }
+  @Mutation
+  public setUnsupportedFeatures(features: any) {
```

Comment:
these were some of the wrong practice we were using in past, but now we should correct it. features should be of specific data type not any. Lets start from here and we will add it for all others in future

Reply:
Added datatype for unsupportedFeatures